dev/io.openliberty.faces.4.0.internal/src/io/openliberty/faces40/internal/fips/FIPSInitializer.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.faces40.internal.fips;
+
+import jakarta.servlet.ServletContainerInitializer;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.ws.common.crypto.CryptoUtils;
+
+public class FIPSInitializer implements ServletContainerInitializer {
```

Comment:
Please add a class-level doc describing what we're doing here and why, so it is obvious in the future.